backend/src/apiserver/client/subject_access_review_fake.go

```diff
@@ -0,0 +1,69 @@
+// Copyright © 2020 Arrikto Inc.  All Rights Reserved.
```

Comment:
It's okay to keep Arrikto copyright, but please add apache license header to every file.

Reply:
Oops, I forgot this! Thanks for catching it